### PR TITLE
Remove mention of corporate sponsors

### PR DESCRIPTION
**Pull request by asymmetric**
Now that the site is owned by the foundation.

### CODE REVIEW

**Comment by asymmetric** [comment]
The build failure is unrelated.

**Comment by fricklerhandwerk** [comment]
I suppose we should still reflect somehow that all this is largely built on that sponsored work. It will still take long time until the general sponsorship situation can be represented uniformly and I'm not aware of any concrete plans how to do that.

@NixOS/foundation opinions?

**Comment by edolstra** [comment]
We list various former sponsors/funders on https://nixos.org/donate.html under "Acknowledgements".

**Comment by fricklerhandwerk** [comment]
Right, but those are sponsors of the foundation. The acknowledgements here are specific, since it was a substantial effort to get this site into the current state. @domenkozar @aspiwack?

**Comment by aspiwack** [comment]
You can imagine that I'd rather keep our name on the website. I don't think that it's an unfair request to ask that our work be explicitly acknowledged in this way.

**Comment by asymmetric** [comment]
Right, so ideally we would acknowledge past sponsors, but also make it clear the website is not *currently* being sponsored by them.

Any ideas about wording?

Is "Previously sponsored by" a good way of putting it?

**Comment by zmitchell** [comment]
P.S. - Build failing due to a broken link. I suspect an anchor on a GitHub README changed. If it’s still failing after tweaking the anchor, just add it to the list of linkcheck ignore URLs.

**Comment by asymmetric** [comment]
OK in the latest [deployed version](https://593bdf0f.nix-dot-dev.pages.dev/), you can see 3 options:

1. leaving the sponsors at the bottom of the sidebar (doesn't scale)
2. adding a sponsors page under acknowledgments (makes sense if we find other uses for the acknowledgments category, maybe past maintainers and Domen?)
1. In the footer, linking to the acknowledgments page

Let me know what you think, I'm inclined to remove 1. (as it doesn't scale with the number of sponsors) and go for 2. and/or 3.

**Comment by thufschmitt** [comment]
I think 2. is a good solution :+1:

**Comment by asymmetric** [comment]
Should we have a way for intersted sponsors to contact us? If we do, I can add it to the [sponsors page](https://fe650ca1.nix-dot-dev.pages.dev/acknowledgments/sponsors).

**Comment by fricklerhandwerk** [comment]
Yes, let's link to the open collective page there.

File: source/acknowledgments/sponsors.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # Sponsors
 
+The following companies have contributed to make this effort possible:
+
 * [Cachix](https://cachix.org) - binary cache hosting
 * [Tweag](https://tweag.io)
+
+You can support the Nix documentation project via our
+[OpenCollective](https://opencollective.com/nixos/projects/documentation-project).
```
Comment (fricklerhandwerk) [inline source/acknowledgments/sponsors.md:9]:
```suggestion
You can support the Nix documentation project via [Open Collective](https://opencollective.com/nixos/projects/documentation-project).
```

**Review by fricklerhandwerk — APPROVED** [review]

**Pull request merged**